src/main/java/frc/robot/Subsystems/WincherClimber.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.Subsystems;
+
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+
+public class WincherClimber extends MotoredGenericSubsystem {
+    private static final double MIN_SPEED = -0.6;
+    private static final double MAX_SPEED = 0.6;
+    private Level magnetLevel = Level.LOWER;
+    private static WincherClimber instance;
+    private final DigitalInput HallEffect;
```

Comment:
variable names are in lowerCamelCase, this variable should be called `hallEffect`instead of `HallEffect`